### DISCUSSION TITLE
强大的Stream API以及并行流与串行流

### DISCUSSION THREAD

**Discussion by giscus** [loveminimal/comment#25] [General]
# 强大的Stream API以及并行流与串行流

强大的Stream API以及并行流与串行流 - https://www.debugdog.cn/%E5%BC%BA%E5%A4%A7%E7%9A%84stream-api%E4%BB%A5%E5%8F%8A%E5%B9%B6%E8%A1%8C%E6%B5%81%E4%B8%8E%E4%B8%B2%E8%A1%8C%E6%B5%81/

https://www.debugdog.cn/%E5%BC%BA%E5%A4%A7%E7%9A%84stream-api%E4%BB%A5%E5%8F%8A%E5%B9%B6%E8%A1%8C%E6%B5%81%E4%B8%8E%E4%B8%B2%E8%A1%8C%E6%B5%81/

**Comment by wangwren** [comment]
测试评论
